# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLTypes.h
# Commit: 16d84439756f
# Full Hash: 16d84439756fae6ab67f7e874dd61d03d3b28905
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r bbfe23c61add -r 16d84439756f dom/canvas/WebGLTypes.h
--- a/dom/canvas/WebGLTypes.h	Mon Jun 08 20:34:15 2020 +0000
+++ b/dom/canvas/WebGLTypes.h	Mon Jun 08 20:36:14 2020 +0000
@@ -990,26 +990,6 @@
 }
 
 // -
-// ClientWebGLContext exports!
-// (So that you don't need to include the mess that is ClientWebGLContext.h)
-namespace gfx {
-class SourceSurface;
-}  // namespace gfx
-
-namespace layers {
-class KnowsCompositor;
-class SurfaceDescriptor;
-}  // namespace layers
-
-namespace webgl {
-
-void Present(ClientWebGLContext&);
-Maybe<layers::SurfaceDescriptor> GetFrontBuffer(ClientWebGLContext&,
-                                                layers::KnowsCompositor*);
-
-}  // namespace webgl
-
-// -
 
 }  // namespace mozilla
 